### DISCUSSION TITLE
Permission role in flow

### DISCUSSION THREAD

**Discussion by mhdaouas** [directus/directus#14692] [Archive]
I still don't understand the permission system in flow nodes. I need to make a conditional update so I created an "edit operation" node and in the permissions field I selected "Other" and entered the role of the only user that is allowed to make that update but I still can update with other users. I tried to write the tole UUID in the ID field but still the same problem.

**Comment by rijkvanzanten** [comment]
> and entered the role of the only user that is allowed to make that update but I still can update with other users. I tried to write the tole UUID in the ID field but still the same problem.

What do you mean by _can update with other users_? The permissions configuration configures what permissions the update operation itself uses, not what "user" is allowed to run that operation

**Reply by mhdaouas** [reply]
And how can I control which user can perform that update?

**Reply by mhdaouas** [reply]
From the Roles & Permissions page?

**Reply by rijkvanzanten** [reply]
I'm not quite sure what the question is. If you want to prevent what things the user can update, that would be Roles & Permissions, yeah
